TourGuide_Horde/25_Southern_Barrens.lua

```diff
@@ -5,7 +5,7 @@ T Ishamuhale |QID|882|
 A Enraged Thunder Lizards |QID|907| |PRE|Ishamuhale|
 
 h Camp Taurajo
-A A New Ore Sample |QID|1153| |N|You need to be level 25 to get this|
+A A New Ore Sample |LV|25| |QID|1153| |N|You need to be level 25 to get this|
```

Comment:
So this isn't how |LV| operates. You had it correct in the other change, but not here. To clarify, |LV| is used to auto-complete the step when you reach or exceed the level provided. In this case, we are looking to **present** the quest based on level criteria. This is my intention to implement with a new tag, like |RLV| for required level. This is not yet implemented, but here would be a good spot to use it. Then, later on we can re-introduce the quest without the |RLV| tag, with the understanding that the player should certainly be able to now accept/complete the quest.

Reply:
I understand now. Can I use |RLV| for those steps for now even if it is not implemented yet or would that cause issues?

Reply:
Yes, as long as we also find a spot to place it without the tag, that makes sense when we know for sure the player is 25+.